scaffolds/java/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use this image as a base
+FROM niaquinto/gradle
+MAINTAINER @toch
+
+# In case someone loses the Dockerfile
+RUN rm -rf /etc/Dockerfile
+ADD Dockerfile /etc/Dockerfile
```

Comment:
The only reason I see is the following: we have the docker image but not the Dockerfile that helped to build it. You can then retrieve it from the docker image.

It's something suggested by the maintainer's base image. What do you think? I don't have opinion about that.